freqtrade/rpc/telegram.py

```diff
@@ -496,6 +498,66 @@ def _forcebuy(self, update: Update, context: CallbackContext) -> None:
         except RPCException as e:
             self._send_msg(str(e))
 
+    @authorized_only
+    def _delete(self, update: Update, context: CallbackContext) -> None:
+        """
+        Handler for /delete <id>.
+        Delete the given trade
+        :param bot: telegram bot
+        :param update: message update
+        :return: None
+        """
+
+        trade_id = context.args[0] if len(context.args) > 0 else None
+        try:
+            msg = self._rpc_delete(trade_id)
+            self._send_msg('Delete Result: `{result}`'.format(**msg))
+
+        except RPCException as e:
+            self._send_msg(str(e))
+
+    @authorized_only
+    def _trades(self, update: Update, context: CallbackContext) -> None:
+        """
+        Handler for /trades <n>
+        Returns last n recent trades.
+        :param bot: telegram bot
+        :param update: message update
+        :return: None
+        """
+        stake_cur = self._config['stake_currency']
+        fiat_disp_cur = self._config.get('fiat_display_currency', '')
+        try:
+            nrecent = int(context.args[0])
+        except (TypeError, ValueError, IndexError):
+            nrecent = 10
+        try:
+            trades = self._rpc_trade_history(
+                nrecent
+            )
+            trades_tab = tabulate(
+                [[trade['open_date'],
+                  trade['pair'],
+                  f"{trade['open_rate']}",
+                  f"{trade['stake_amount']}",
+                  trade['close_date'],
+                  f"{trade['close_rate']}",
+                  f"{trade['close_profit_abs']}"] for trade in trades['trades']],
+                headers=[
+                    'Open Date',
+                    'Pair',
+                    'Open rate',
+                    'Stake Amount',
```

Comment:
I'm not certain if we should use Stake-Amount, or rather "amount"? 
usage of stake-amount is only usefull in the edge-case where you set stake_amount to unlimited, otherwise it'll always be "X".